scripts/mirror.test.js

```diff
@@ -0,0 +1,141 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+'use strict';
+
+const assert = require('assert').strict;
+
+const { browsers } = require('..');
+const { getMatchingBrowserVersion } = require('./mirror');
+
+describe('mirror', function () {
+  const match = (source_browser, source_version, dest_browser) => {
+    const source_release = browsers[source_browser].releases[source_version];
+    return getMatchingBrowserVersion(dest_browser, source_release);
+  };
+
+  it('chrome_android', function () {
+    assert.equal(match('chrome', '1', 'chrome_android'), '18');
+    assert.equal(match('chrome', '2', 'chrome_android'), '18');
+    assert.equal(match('chrome', '17', 'chrome_android'), '18');
+    assert.equal(match('chrome', '18', 'chrome_android'), '18');
+    assert.equal(match('chrome', '19', 'chrome_android'), '25');
+    assert.equal(match('chrome', '24', 'chrome_android'), '25');
+    assert.equal(match('chrome', '25', 'chrome_android'), '25');
+    assert.equal(match('chrome', '26', 'chrome_android'), '26');
+  });
+
+  it('edge', function () {
+    assert.equal(match('chrome', '1', 'edge'), false);
+    assert.equal(match('chrome', '27', 'edge'), false);
+    assert.equal(match('chrome', '28', 'edge'), '79');
+    assert.equal(match('chrome', '79', 'edge'), '79');
+    assert.equal(match('chrome', '80', 'edge'), '80');
+  });
+
+  it('firefox_android', function () {
+    assert.equal(match('firefox', '1', 'firefox_android'), '4');
+    assert.equal(match('firefox', '2', 'firefox_android'), '4');
+    assert.equal(match('firefox', '3', 'firefox_android'), '4');
+    assert.equal(match('firefox', '4', 'firefox_android'), '4');
+    assert.equal(match('firefox', '68', 'firefox_android'), '68');
+    assert.equal(match('firefox', '69', 'firefox_android'), '79');
+    assert.equal(match('firefox', '79', 'firefox_android'), '79');
+    assert.equal(match('firefox', '80', 'firefox_android'), '80');
+  });
+
+  it('opera', function () {
+    assert.equal(match('chrome', '1', 'opera'), '15');
+    assert.equal(match('chrome', '27', 'opera'), '15');
+    assert.equal(match('chrome', '28', 'opera'), '15');
+    assert.equal(match('chrome', '29', 'opera'), '16');
+    assert.equal(match('chrome', '30', 'opera'), '17');
+  });
+
+  it('opera_android', function () {
+    assert.equal(match('chrome_android', '18', 'opera_android'), '14');
+    assert.equal(match('chrome_android', '26', 'opera_android'), '14');
+    assert.equal(match('chrome_android', '27', 'opera_android'), '14'); // wrong, should be 15
```

Comment:
This is wrong because Opera Android 14 uses Blink 26 as the engine, but there's no such thing. Chrome Android 26 used WebKit 537.31.